### ISSUE TITLE
Nejistota při mazání pojmů v OG

### ISSUE THREAD

**Issue by jakubklimek** [datagov-cz/sgov-assembly-line#176]
# Popis
Při mazání pojmů z OG cítím nejistotu, zda něco nepokazím, konkrétně zda nesmažu pojem z cizího slovníku.
Jde o to, že jak u položek ze slovníku k editaci, tak u položek přidaného slovníku ke čtení je stejný symbol popelnice při mazání. To mi říká, že to bude stejná akce. Jenže není. A to se dozvím až v dialogu, když se na popelnici odvážím kliknout - což je pozdě.

Rozdíl tedy zřejmě poznám podle symbolu před slovníkem (tužka nebo dokument), ale tím že tam není ani žádný tooltip, tak si nejsem jistý, že to je právě toto rozlišení.

![mazání pojmů](https://github.com/datagov-cz/sgov-assembly-line/assets/718827/3471a9af-f2fd-473e-9184-4d73f2dbfef0)

# Užitečnost
2. zmíněný scénář lze bez funkce realizovat, ale zdržuje mě to

Navrhoval bych přidat vysvětlující tooltipy, které se zobrazí před kliknutím na popelnici, které budou jasně rozlišovat, co se bude dít. Dále bych takové tooltipy přidal i k symbolům před slovníky, aby to vysvětlilo, že některý slovník je k editaci a některý pouze ke čtení.

**Comment by MichalMed** [comment]
souhlasím, že je to matoucí, popelnici u slovníku ke čtení bych možná úplně odstranil (nejsem si jistý, nakolik je tahle možnost vůbec využívatelná)

**Comment by bindeali** [comment]
Mělo by být vyřešeno na PROD, prosím o kontrolu. Ikony a popisky jsou změněny a při najetí na "ikonu" slovníku se objeví vysvětlující tooltip.

**Comment by jakubklimek** [comment]
Ikonky mi přijdou OK, jen "odstranit z pracovního prostoru" nevím zda je pochopitelné, že to nesmaže ze slovníku. To ale už nechám na diskuzi, stačilo by mi to i takto.

**Comment by MichalMed** [comment]
jen připomínám, že nic jako "pracovní prostor" neexistuje, proto bych se přimlouval za jiný tooltip.
